aea/cli/generate.py

```diff
@@ -38,7 +38,7 @@
     PublicId,
 )
 from aea.configurations.loader import ConfigLoader
-from aea.protocols.generator import ProtocolGenerator
+from aea.protocols.generator.generator import ProtocolGenerator
```

Comment:
Your init file below is missing header info

Reply:
Done